pytest_django/plugin.py

```diff
@@ -332,11 +336,21 @@ def django_test_environment(request):
     """
     if django_settings_is_configured():
         _setup_django()
+        from distutils.version import StrictVersion
+        import django
         from django.conf import settings as dj_settings
         from django.test.utils import (setup_test_environment,
                                        teardown_test_environment)
-        dj_settings.DEBUG = False
-        setup_test_environment()
+        if request.config.getvalue('djangodebugmode') != 'None':
+            django_debug_mode = bool(strtobool(request.config.getvalue('djangodebugmode')))
+            if StrictVersion(django.get_version()) >= StrictVersion('1.11'):
```

Comment:
I think this should use the same mechanism for comparing versions like we do elsewhere, i.e. just via `django.VERSION`.

Reply:
Done.